apps/src/music/views/ProgressSlider.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {memo} from 'react';
+import moduleStyles from './progress-slider.module.scss';
+
+interface ProgressSliderProps {
+  percentProgress: number;
+}
+
+/*
+  * A progress slider that displays the current progress of a music lab song.
+  * The slider is a gray bar that fills up with a white color as the song progresses.
+  * A playhead moves along the slider to show the current progress of the song.
+  * @param percentProgress - The percentage of the song that has been played.
+  * @param sliderWidth - The width of the slider in pixels.
```

Comment:
nit: is `sliderWidth` meant to be an optional prop as well? (or if not, can we remove it from this doc?)

Reply:
Thanks for catching! I originally had it as a prop, but removed and forgot to update comment. Will update!